include/fdreadoutlibs/tpg/TPGConstants.hpp

```diff
@@ -14,12 +14,12 @@
 
 namespace swtpg {
 
-const constexpr std::size_t SUPERCHUNK_FRAME_SIZE = 5568; // for 12: 5568  for 6: 2784
+const constexpr std::size_t SUPERCHUNK_FRAME_SIZE = 5664;//5568; // for 12: 5568  for 6: 2784
```

Comment:
Moving this to a global TPG namespace would break the assumption on the Proto-WIB's software TPG implementation, no?